docs/Manual/Plugins/General/ConvertProfile.html

```diff
@@ -344,7 +344,7 @@ <h1 id="#convert_profile" class="anchor" >convert_profile</h1>
 particular software version. For example, Rekall requires a Linux Kernel profile
 to be able to analyze a memory image of the Linux kernel.</p>
 <p>The <code>convert_profile</code> plugin converts profiles other formats to the standard
-JSON format used by Relall. There are two main use cases:</p>
+JSON format used by Rekall. There are two main use cases:</p>
```

Comment:
No problem - the changes will be lost the next time the code compiles :-).

You can just compile the html yourself by checking out the gh-pages branch and running the script:

```
(Dev)⟫ python scripts/gen_site.py .
Converting ./Discuss.md
Converting ./releases.md
Converting ./github.md
Converting ./docs.md
```

and you can serve the site locally:
```
(Dev)⟫ python scripts/gen_site.py --serve .
Serving HTTP on 127.0.0.1 port 8000 ...
```